process.cabal

```diff
@@ -66,15 +68,17 @@ library
     else
         c-sources:
             cbits/posix/runProcess.c
+            cbits/posix/bsd_closefrom.c
             cbits/posix/fork_exec.c
             cbits/posix/posix_spawn.c
             cbits/posix/find_executable.c
+        includes:
+            runProcess.h
+            cbits/posix/bsd_closefrom.h
```

Comment:
I think you want to shorten this to `bsd_closefrom.h` and add `cbits/posix` to `include-dirs`.

Reply:
Done, I just resubmitted this to test in the GHC pipelines